### PR TITLE
feat: new method to fetch makes suggestions

### PR DESCRIPTION
**Pull request by talamcol**
References (hackdays)

## Motivation and context

for the full text search recommendations, backend introduced a new API

Bildschirmfoto 2021-09-07 um 14 04 42
(the part on the left)

### CODE REVIEW

**Review by Averethel — APPROVED** [review]

**Review by ufkaya — APPROVED** [review]
LGTM
Can you add a test?

**Comment by talamcol** [comment]
> LGTM
> Can you add a test?

what do you want to test? since I'm neither doing something with the query nor with the response I'm not sure what I should test

**Comment by ufkaya** [comment]
> > LGTM
> > Can you add a test?
> 
> what do you want to test? since I'm neither doing something with the query nor with the response I'm not sure what I should test

Usually, we try to add a test, even if it is a simple one, you can check `currentMakeModels.test.ts`, up to you

File: src/services/search/currentMakeModels.ts [hunk 1]
```diff
@@ -19,3 +19,15 @@ export const fetchCurrentModels = async ({
...
 }
+
+export const fetchMakesSuggestions = async ({
+  query,
+  options = {},
+}: {
+  query: string
+  options?: ApiCallOptions
+}): Promise<MakeWithModels[]> =>
+  await fetchPath({
```
Comment (lkappeler) [inline src/services/search/currentMakeModels.ts:30]: I guess you don't wanna await that rather return it?

Comment (talamcol) [reply src/services/search/currentMakeModels.ts:30]: both works, will change it. Since the function is declared as async, you have to await it anyway

Comment (lkappeler) [reply src/services/search/currentMakeModels.ts:30]: Oh, I missed that you used a shorthand arrow function on multiple lines...

**Pull request merged**